### DISCUSSION TITLE
Arching Velocity Inlet in Free Slip BC

### DISCUSSION THREAD

**Discussion by dyat29** [xcompact3d/Incompact3d#279] [Q&A] [answered]
Dear All,

I am currently working on adjusting the boundary condition of a channel flow to a free slip condition at the top wall(nclyn = 1 in input.i3d). However, I observed that the inlet velocity profile is still in elliptical-shaped, which suggests that the top wall still exhibits frictional effects. It is not zero, though, but is there a reason why it looks like this? Thanks!

Mike
![Screenshot 2024-07-11 051111](https://github.com/xcompact3d/Incompact3d/assets/175279110/024b2b65-c7a7-4df3-8759-50d2701fa8d4)

**Comment by slaizet** [comment, marked as answer]
Hi Mike,
What is your initial condition? If it is the same as a "normal" channel then you would have to wait for quite some time for the profile to change.
Best,
Sylvain
